app/assets/javascripts/pretty-text/addon/allow-lister.js

```diff
@@ -156,6 +156,7 @@ export const DEFAULT_LIST = [
   "blockquote",
   "br",
   "code",
+  "code[class]",
```

Comment:
we can't allow for injection of class, people will use it for abuse. We need to inject a data attribute instead... maybe `data-lang` or something like that.

Reply:
The markdown spec seems to expect `language-ruby` (which we convert to `lang-ruby` for legacy reasons I guess. So should I deviate from the spec in this case, and when not in the list of accepted classes go with `data-block="mermaid"` ?